<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectTargetForm</class>
 <widget class="QDialog" name="SelectTargetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Target mass number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="MassNumb">
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>28</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Target element number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ElemNumb">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>14</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Target width(mg/cm^2):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Width">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>4.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Unit:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mgcm2Button">
         <property name="text">
          <string>mg/cm^2</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="gcm2Button">
         <property name="text">
          <string>g/cm^2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="umButton">
         <property name="text">
          <string>um</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>label_4</zorder>
  <zorder>mgcm2Button</zorder>
  <zorder>gcm2Button</zorder>
  <zorder>umButton</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectTargetForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectTargetForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
